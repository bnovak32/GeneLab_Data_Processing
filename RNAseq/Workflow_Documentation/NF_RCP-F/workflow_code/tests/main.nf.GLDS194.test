nextflow_pipeline {

    name "Test Workflow main.nf"
    script "main.nf"

    test("GLDS-194:Mouse,PairedEnd,ERCC: Should run without failures") {

        when {
            params {
                gldsAccession = 'GLDS-194' // GeneLab Data Accession Number, e.g. GLDS-104
                use_dummy_gene_counts = true // Use random gene counts for Deseq2, this addresses an issue where low/zero gene counts causes DGE analysis to fail
                genomeSubsample = 19 // Subsamples the reference fasta and gtf to a single sequence (often representing a single chromosome)
                truncateTo = 300 // Subsamples the raw reads files to the specified number of reads for EACH raw reads file.
            }
        }

        then {
            def TEST_DATASET = "GLDS-194"
            assert workflow.success
            assert snapshot(
                            path("${params.outputDir}/GLDS-194/Metadata").list(),
                            path("${params.outputDir}/GLDS-194/00-RawData/Fastq").list(),
                            path("${params.outputDir}/GLDS-194/01-TG_Preproc/Fastq").list(),
                            path("${params.outputDir}/${ TEST_DATASET }/02-STAR_Alignment/STAR_NumNonZeroGenes.csv"),
                            path("${params.outputDir}/${ TEST_DATASET }/02-STAR_Alignment/STAR_Unnormalized_Counts.csv"),
                            path("${params.outputDir}/GLDS-194/03-RSEM_Counts/Mmus_BAL-TAL_LRTN_BSL_Rep1_B7.genes.results"),
                            path("${params.outputDir}/GLDS-194/03-RSEM_Counts/Mmus_BAL-TAL_LRTN_BSL_Rep1_B7.isoforms.results"),
                            path("${params.outputDir}/GLDS-194/03-RSEM_Counts/RSEM_NumNonZeroGenes.csv"),
                            path("${params.outputDir}/GLDS-194/03-RSEM_Counts/RSEM_Unnormalized_Counts.csv"),
                            path("${params.outputDir}/GLDS-194/04-DESeq2_NormCounts").list(),
                            path("${params.outputDir}/GLDS-194/05-DESeq2_DGE").list(),
                            path("${params.outputDir}/GLDS-194/GeneLab").list(),
                            path("${params.outputDir}/GLDS-194/RSeQC_Analyses/02_geneBody_coverage/Mmus_BAL-TAL_LRTN_BSL_Rep1_B7/Mmus_BAL-TAL_LRTN_BSL_Rep1_B7.geneBodyCoverage.txt"),
                            path("${params.outputDir}/GLDS-194/RSeQC_Analyses/03_infer_experiment/Mmus_BAL-TAL_LRTN_GC_Rep3_G9_infer_expt.out"),
                            path("${params.outputDir}/GLDS-194/RSeQC_Analyses/04_inner_distance/Mmus_BAL-TAL_LRTN_FLT_Rep2_F7/Mmus_BAL-TAL_LRTN_FLT_Rep2_F7.inner_distance.txt"),
                            path("${params.outputDir}/GLDS-194/RSeQC_Analyses/05_read_distribution/Mmus_BAL-TAL_RRTN_BSL_Rep4_B10_read_dist.out"),
                            ).match()
        }
    }
}
