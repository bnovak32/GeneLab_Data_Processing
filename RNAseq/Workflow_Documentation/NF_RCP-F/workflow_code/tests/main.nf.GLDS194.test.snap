{
    "GLDS-194: Should run without failures": {
        "content": [
            [
                "GLDS-194_bulkRNASeq_v1_runsheet.csv:md5,baf0d7e3b09aaa2e3bfc29a51f141324",
                "GLDS-194_metadata_GLDS-194-ISA.zip:md5,e19f03c4800afc9d93cbfb3d4c5ddcbd",
                "GLDS-194_metadata_table.txt:md5,0787b35bcebd9f8bf129ea40c6b7852b"
            ],
            [
                "Mmus_BAL-TAL_LRTN_BSL_Rep1_B7_R1_raw.fastq.gz:md5,6a73b8d73fdb00701a4cd7300e819ce6",
                "Mmus_BAL-TAL_LRTN_BSL_Rep1_B7_R2_raw.fastq.gz:md5,246d5d934b465d5444aa72971f3681eb",
                "Mmus_BAL-TAL_LRTN_FLT_Rep1_F6_R1_raw.fastq.gz:md5,5e72bd4424665bed0f7b1089e4a52462",
                "Mmus_BAL-TAL_LRTN_FLT_Rep1_F6_R2_raw.fastq.gz:md5,130975a783f319ea7e18dddb900b8f38",
                "Mmus_BAL-TAL_LRTN_FLT_Rep2_F7_R1_raw.fastq.gz:md5,5646fd1cedf996d15b3a65afea9cd9e0",
                "Mmus_BAL-TAL_LRTN_FLT_Rep2_F7_R2_raw.fastq.gz:md5,397b927eb309badedff206e53ad30daa",
                "Mmus_BAL-TAL_LRTN_FLT_Rep3_F8_R1_raw.fastq.gz:md5,31c10bf2680020dcafdc20be99ed0ec0",
                "Mmus_BAL-TAL_LRTN_FLT_Rep3_F8_R2_raw.fastq.gz:md5,e0a6cd1965bd50587e4854d65a57c415",
                "Mmus_BAL-TAL_LRTN_FLT_Rep4_F9_R1_raw.fastq.gz:md5,a299128e3f3220f89a68374dd1f96153",
                "Mmus_BAL-TAL_LRTN_FLT_Rep4_F9_R2_raw.fastq.gz:md5,501f416e5d4d344f145ade9637b9662e",
                "Mmus_BAL-TAL_LRTN_FLT_Rep5_F10_R1_raw.fastq.gz:md5,888eef1d5b18cff692ee317ab10085fc",
                "Mmus_BAL-TAL_LRTN_FLT_Rep5_F10_R2_raw.fastq.gz:md5,17ce3a33e6bc828e10ef89aad0c11b04",
                "Mmus_BAL-TAL_LRTN_GC_Rep1_G6_R1_raw.fastq.gz:md5,45591253c449eec31daa9c6a205dae9a",
                "Mmus_BAL-TAL_LRTN_GC_Rep1_G6_R2_raw.fastq.gz:md5,b2ca58ca347e97d26c002ebb55cec064",
                "Mmus_BAL-TAL_LRTN_GC_Rep2_G8_R1_raw.fastq.gz:md5,aff030ff1aac5f2cfc62d54069256f9c",
                "Mmus_BAL-TAL_LRTN_GC_Rep2_G8_R2_raw.fastq.gz:md5,b29353231a016036c3b4dc7f9c6afff0",
                "Mmus_BAL-TAL_LRTN_GC_Rep3_G9_R1_raw.fastq.gz:md5,617ba61ce336a913d5d91d335a47b29a",
                "Mmus_BAL-TAL_LRTN_GC_Rep3_G9_R2_raw.fastq.gz:md5,449b0a1adc1178541e44cf8d4f21ea35",
                "Mmus_BAL-TAL_RRTN_BSL_Rep2_B8_R1_raw.fastq.gz:md5,098808ac73e76368db215d1ba8b200dc",
                "Mmus_BAL-TAL_RRTN_BSL_Rep2_B8_R2_raw.fastq.gz:md5,17dbf5a1c4f2eced1f1441ff6f288d65",
                "Mmus_BAL-TAL_RRTN_BSL_Rep3_B9_R1_raw.fastq.gz:md5,d893bb2f23f8857a41d02d93b09dfa24",
                "Mmus_BAL-TAL_RRTN_BSL_Rep3_B9_R2_raw.fastq.gz:md5,4e76bdfbe4fc04ff1b90e62e79be8b3f",
                "Mmus_BAL-TAL_RRTN_BSL_Rep4_B10_R1_raw.fastq.gz:md5,dd0ad4670c30a4441660cf5c17eb0a1f",
                "Mmus_BAL-TAL_RRTN_BSL_Rep4_B10_R2_raw.fastq.gz:md5,ad84dadec29cb1cd7e8d9b2797cf2e52",
                "Mmus_BAL-TAL_RRTN_GC_Rep4_G10_R1_raw.fastq.gz:md5,731c77e2a63594542ab813a32d5a740c",
                "Mmus_BAL-TAL_RRTN_GC_Rep4_G10_R2_raw.fastq.gz:md5,4ebb24a71d6b343ea3409b1495250005"
            ],
            [
                "Mmus_BAL-TAL_LRTN_BSL_Rep1_B7_R1_trimmed.fastq.gz:md5,b04895bf9ad3fc01d7d9a397b6c66dc3",
                "Mmus_BAL-TAL_LRTN_BSL_Rep1_B7_R2_trimmed.fastq.gz:md5,e7e2d2a923196c2508fbe08f59c52170",
                "Mmus_BAL-TAL_LRTN_FLT_Rep1_F6_R1_trimmed.fastq.gz:md5,ad115d4ed8ef8e8eb6c80ba8e616baae",
                "Mmus_BAL-TAL_LRTN_FLT_Rep1_F6_R2_trimmed.fastq.gz:md5,64696f438d14244117e3ad4c5d9983f0",
                "Mmus_BAL-TAL_LRTN_FLT_Rep2_F7_R1_trimmed.fastq.gz:md5,bb7b364d3c2b7648c9df0994951ebdcc",
                "Mmus_BAL-TAL_LRTN_FLT_Rep2_F7_R2_trimmed.fastq.gz:md5,217c77db3e39d890b59210f369e00518",
                "Mmus_BAL-TAL_LRTN_FLT_Rep3_F8_R1_trimmed.fastq.gz:md5,2718364cd4c80c24e40cb0dda7454a8e",
                "Mmus_BAL-TAL_LRTN_FLT_Rep3_F8_R2_trimmed.fastq.gz:md5,acce60d46824d2398ecef34f1ebb86a7",
                "Mmus_BAL-TAL_LRTN_FLT_Rep4_F9_R1_trimmed.fastq.gz:md5,b9056add48e8ed9e4430a5c33efe35f7",
                "Mmus_BAL-TAL_LRTN_FLT_Rep4_F9_R2_trimmed.fastq.gz:md5,348f72ed4cce2dae61ce1ed32c5aea6b",
                "Mmus_BAL-TAL_LRTN_FLT_Rep5_F10_R1_trimmed.fastq.gz:md5,6d4fa4cab221f61b0b2f40ae25192866",
                "Mmus_BAL-TAL_LRTN_FLT_Rep5_F10_R2_trimmed.fastq.gz:md5,1c87e46892ba9f6ee5d6facf91555548",
                "Mmus_BAL-TAL_LRTN_GC_Rep1_G6_R1_trimmed.fastq.gz:md5,aa1a90ff76a418d733493f0c7bb72648",
                "Mmus_BAL-TAL_LRTN_GC_Rep1_G6_R2_trimmed.fastq.gz:md5,85adc9eb26785744a685ae8570389f0a",
                "Mmus_BAL-TAL_LRTN_GC_Rep2_G8_R1_trimmed.fastq.gz:md5,2e9660ecbaac210effc06e35511b4898",
                "Mmus_BAL-TAL_LRTN_GC_Rep2_G8_R2_trimmed.fastq.gz:md5,acfaea899dfcf9f114b9b3398a779e99",
                "Mmus_BAL-TAL_LRTN_GC_Rep3_G9_R1_trimmed.fastq.gz:md5,90d0476253287e7fbab912f2d8136dcc",
                "Mmus_BAL-TAL_LRTN_GC_Rep3_G9_R2_trimmed.fastq.gz:md5,a92ad5b94ba6f2fb3d8cc4556f9bff95",
                "Mmus_BAL-TAL_RRTN_BSL_Rep2_B8_R1_trimmed.fastq.gz:md5,f8666de8ec5e5a5b2baba81f96cdc9e2",
                "Mmus_BAL-TAL_RRTN_BSL_Rep2_B8_R2_trimmed.fastq.gz:md5,ad0a8b147859ec415318d60dea056a38",
                "Mmus_BAL-TAL_RRTN_BSL_Rep3_B9_R1_trimmed.fastq.gz:md5,e169eb48e74c05e0ae4e7aaeb6175c5a",
                "Mmus_BAL-TAL_RRTN_BSL_Rep3_B9_R2_trimmed.fastq.gz:md5,b0b5512e41d87ce28c8b58cdcf38a88f",
                "Mmus_BAL-TAL_RRTN_BSL_Rep4_B10_R1_trimmed.fastq.gz:md5,d31b2a4a3f9722430d01a9728ee2c857",
                "Mmus_BAL-TAL_RRTN_BSL_Rep4_B10_R2_trimmed.fastq.gz:md5,0c688523916b3dbe3bdf96bd32aa1968",
                "Mmus_BAL-TAL_RRTN_GC_Rep4_G10_R1_trimmed.fastq.gz:md5,d68d80608036afc6adcbfff4704eef89",
                "Mmus_BAL-TAL_RRTN_GC_Rep4_G10_R2_trimmed.fastq.gz:md5,880d7cf7a8a23c16e45370cf8c8f78a7"
            ],
            "Mmus_BAL-TAL_LRTN_BSL_Rep1_B7_Aligned.sortedByCoord_sorted.out.bam:md5,be3d2dd16454061a43d70d46b1465774",
            "Mmus_BAL-TAL_LRTN_BSL_Rep1_B7.genes.results:md5,ed4ca7a2d1352f82e0e97491212d40fa",
            "Mmus_BAL-TAL_LRTN_BSL_Rep1_B7.isoforms.results:md5,1a519ec4f3e8e65a6aabd05c298cd5d7",
            "RSEM_NumNonZeroGenes.csv:md5,866c1d9110c36a7ec9ee1eac93d3a1a3",
            "RSEM_Unnormalized_Counts.csv:md5,4aa50a767781e24ba1abed03adeda0e7",
            [
                "ERCC_Normalized_Counts.csv:md5,b1d9d5a546a23b6709a9f8c60548b6a7",
                "Normalized_Counts.csv:md5,b4ba348d5446f8ba546a46b966087c1b",
                "RSEM_Unnormalized_Counts.csv:md5,931c6070b5e19909929c5a217713500b"
            ],
            [
                [
                    "ERCCnorm_SampleTable.csv:md5,bfd18bbc7d34e41c23f0c9107f5d75c9",
                    "ERCCnorm_contrasts.csv:md5,66d74b686885ffd4eccdf55823c0e3ce",
                    "ERCCnorm_differential_expression.csv:md5,e713875ee762bccaacac2264f215d8b1",
                    "visualization_PCA_table_ERCCnorm.csv:md5,85edbf46c6af087cfe83e3a60f069a71",
                    "visualization_output_table_ERCCnorm.csv:md5,245d2ae218045f8c0a519c3c68119c73"
                ],
                "SampleTable.csv:md5,bfd18bbc7d34e41c23f0c9107f5d75c9",
                "contrasts.csv:md5,66d74b686885ffd4eccdf55823c0e3ce",
                "differential_expression.csv:md5,00cf45e546529c81c0a43ae1b8495a59",
                "visualization_PCA_table.csv:md5,5c461d35b12d5946c2105f705a03c6d3",
                "visualization_output_table.csv:md5,58578caedc33e6a0230ba80abe61f0d5"
            ],
            [
                "software_versions.md:md5,9f59704a831ca5fba8383562d8e553d7"
            ],
            "Mmus_BAL-TAL_LRTN_BSL_Rep1_B7.geneBodyCoverage.txt:md5,2bd21b4fa65d24275adab0e1273af337",
            "Mmus_BAL-TAL_LRTN_GC_Rep3_G9_infer_expt.out:md5,248ae9ba2b87a705ea046389342942a1",
            "Mmus_BAL-TAL_LRTN_FLT_Rep2_F7.inner_distance.txt:md5,f3984c71448fc339a5c1e8773dd67c34",
            "Mmus_BAL-TAL_RRTN_BSL_Rep4_B10_read_dist.out:md5,becb7190c85474e7f3ef07c6372491f6"
        ],
        "timestamp": "2023-01-25T23:04:09+0000"
    },
    "GLDS-194:Mouse,PairedEnd,ERCC: Should run without failures": {
        "content": [
            [
                "GLDS-194_bulkRNASeq_v1_runsheet.csv:md5,baf0d7e3b09aaa2e3bfc29a51f141324",
                "GLDS-194_metadata_GLDS-194-ISA.zip:md5,e19f03c4800afc9d93cbfb3d4c5ddcbd",
                "GLDS-194_metadata_table.txt:md5,0787b35bcebd9f8bf129ea40c6b7852b"
            ],
            [
                "Mmus_BAL-TAL_LRTN_BSL_Rep1_B7_R1_raw.fastq.gz:md5,6a73b8d73fdb00701a4cd7300e819ce6",
                "Mmus_BAL-TAL_LRTN_BSL_Rep1_B7_R2_raw.fastq.gz:md5,246d5d934b465d5444aa72971f3681eb",
                "Mmus_BAL-TAL_LRTN_FLT_Rep1_F6_R1_raw.fastq.gz:md5,5e72bd4424665bed0f7b1089e4a52462",
                "Mmus_BAL-TAL_LRTN_FLT_Rep1_F6_R2_raw.fastq.gz:md5,130975a783f319ea7e18dddb900b8f38",
                "Mmus_BAL-TAL_LRTN_FLT_Rep2_F7_R1_raw.fastq.gz:md5,5646fd1cedf996d15b3a65afea9cd9e0",
                "Mmus_BAL-TAL_LRTN_FLT_Rep2_F7_R2_raw.fastq.gz:md5,397b927eb309badedff206e53ad30daa",
                "Mmus_BAL-TAL_LRTN_FLT_Rep3_F8_R1_raw.fastq.gz:md5,31c10bf2680020dcafdc20be99ed0ec0",
                "Mmus_BAL-TAL_LRTN_FLT_Rep3_F8_R2_raw.fastq.gz:md5,e0a6cd1965bd50587e4854d65a57c415",
                "Mmus_BAL-TAL_LRTN_FLT_Rep4_F9_R1_raw.fastq.gz:md5,a299128e3f3220f89a68374dd1f96153",
                "Mmus_BAL-TAL_LRTN_FLT_Rep4_F9_R2_raw.fastq.gz:md5,501f416e5d4d344f145ade9637b9662e",
                "Mmus_BAL-TAL_LRTN_FLT_Rep5_F10_R1_raw.fastq.gz:md5,888eef1d5b18cff692ee317ab10085fc",
                "Mmus_BAL-TAL_LRTN_FLT_Rep5_F10_R2_raw.fastq.gz:md5,17ce3a33e6bc828e10ef89aad0c11b04",
                "Mmus_BAL-TAL_LRTN_GC_Rep1_G6_R1_raw.fastq.gz:md5,45591253c449eec31daa9c6a205dae9a",
                "Mmus_BAL-TAL_LRTN_GC_Rep1_G6_R2_raw.fastq.gz:md5,b2ca58ca347e97d26c002ebb55cec064",
                "Mmus_BAL-TAL_LRTN_GC_Rep2_G8_R1_raw.fastq.gz:md5,aff030ff1aac5f2cfc62d54069256f9c",
                "Mmus_BAL-TAL_LRTN_GC_Rep2_G8_R2_raw.fastq.gz:md5,b29353231a016036c3b4dc7f9c6afff0",
                "Mmus_BAL-TAL_LRTN_GC_Rep3_G9_R1_raw.fastq.gz:md5,617ba61ce336a913d5d91d335a47b29a",
                "Mmus_BAL-TAL_LRTN_GC_Rep3_G9_R2_raw.fastq.gz:md5,449b0a1adc1178541e44cf8d4f21ea35",
                "Mmus_BAL-TAL_RRTN_BSL_Rep2_B8_R1_raw.fastq.gz:md5,098808ac73e76368db215d1ba8b200dc",
                "Mmus_BAL-TAL_RRTN_BSL_Rep2_B8_R2_raw.fastq.gz:md5,17dbf5a1c4f2eced1f1441ff6f288d65",
                "Mmus_BAL-TAL_RRTN_BSL_Rep3_B9_R1_raw.fastq.gz:md5,d893bb2f23f8857a41d02d93b09dfa24",
                "Mmus_BAL-TAL_RRTN_BSL_Rep3_B9_R2_raw.fastq.gz:md5,4e76bdfbe4fc04ff1b90e62e79be8b3f",
                "Mmus_BAL-TAL_RRTN_BSL_Rep4_B10_R1_raw.fastq.gz:md5,dd0ad4670c30a4441660cf5c17eb0a1f",
                "Mmus_BAL-TAL_RRTN_BSL_Rep4_B10_R2_raw.fastq.gz:md5,ad84dadec29cb1cd7e8d9b2797cf2e52",
                "Mmus_BAL-TAL_RRTN_GC_Rep4_G10_R1_raw.fastq.gz:md5,731c77e2a63594542ab813a32d5a740c",
                "Mmus_BAL-TAL_RRTN_GC_Rep4_G10_R2_raw.fastq.gz:md5,4ebb24a71d6b343ea3409b1495250005"
            ],
            [
                "Mmus_BAL-TAL_LRTN_BSL_Rep1_B7_R1_trimmed.fastq.gz:md5,b04895bf9ad3fc01d7d9a397b6c66dc3",
                "Mmus_BAL-TAL_LRTN_BSL_Rep1_B7_R2_trimmed.fastq.gz:md5,e7e2d2a923196c2508fbe08f59c52170",
                "Mmus_BAL-TAL_LRTN_FLT_Rep1_F6_R1_trimmed.fastq.gz:md5,ad115d4ed8ef8e8eb6c80ba8e616baae",
                "Mmus_BAL-TAL_LRTN_FLT_Rep1_F6_R2_trimmed.fastq.gz:md5,64696f438d14244117e3ad4c5d9983f0",
                "Mmus_BAL-TAL_LRTN_FLT_Rep2_F7_R1_trimmed.fastq.gz:md5,bb7b364d3c2b7648c9df0994951ebdcc",
                "Mmus_BAL-TAL_LRTN_FLT_Rep2_F7_R2_trimmed.fastq.gz:md5,217c77db3e39d890b59210f369e00518",
                "Mmus_BAL-TAL_LRTN_FLT_Rep3_F8_R1_trimmed.fastq.gz:md5,2718364cd4c80c24e40cb0dda7454a8e",
                "Mmus_BAL-TAL_LRTN_FLT_Rep3_F8_R2_trimmed.fastq.gz:md5,acce60d46824d2398ecef34f1ebb86a7",
                "Mmus_BAL-TAL_LRTN_FLT_Rep4_F9_R1_trimmed.fastq.gz:md5,b9056add48e8ed9e4430a5c33efe35f7",
                "Mmus_BAL-TAL_LRTN_FLT_Rep4_F9_R2_trimmed.fastq.gz:md5,348f72ed4cce2dae61ce1ed32c5aea6b",
                "Mmus_BAL-TAL_LRTN_FLT_Rep5_F10_R1_trimmed.fastq.gz:md5,6d4fa4cab221f61b0b2f40ae25192866",
                "Mmus_BAL-TAL_LRTN_FLT_Rep5_F10_R2_trimmed.fastq.gz:md5,1c87e46892ba9f6ee5d6facf91555548",
                "Mmus_BAL-TAL_LRTN_GC_Rep1_G6_R1_trimmed.fastq.gz:md5,aa1a90ff76a418d733493f0c7bb72648",
                "Mmus_BAL-TAL_LRTN_GC_Rep1_G6_R2_trimmed.fastq.gz:md5,85adc9eb26785744a685ae8570389f0a",
                "Mmus_BAL-TAL_LRTN_GC_Rep2_G8_R1_trimmed.fastq.gz:md5,2e9660ecbaac210effc06e35511b4898",
                "Mmus_BAL-TAL_LRTN_GC_Rep2_G8_R2_trimmed.fastq.gz:md5,acfaea899dfcf9f114b9b3398a779e99",
                "Mmus_BAL-TAL_LRTN_GC_Rep3_G9_R1_trimmed.fastq.gz:md5,90d0476253287e7fbab912f2d8136dcc",
                "Mmus_BAL-TAL_LRTN_GC_Rep3_G9_R2_trimmed.fastq.gz:md5,a92ad5b94ba6f2fb3d8cc4556f9bff95",
                "Mmus_BAL-TAL_RRTN_BSL_Rep2_B8_R1_trimmed.fastq.gz:md5,f8666de8ec5e5a5b2baba81f96cdc9e2",
                "Mmus_BAL-TAL_RRTN_BSL_Rep2_B8_R2_trimmed.fastq.gz:md5,ad0a8b147859ec415318d60dea056a38",
                "Mmus_BAL-TAL_RRTN_BSL_Rep3_B9_R1_trimmed.fastq.gz:md5,e169eb48e74c05e0ae4e7aaeb6175c5a",
                "Mmus_BAL-TAL_RRTN_BSL_Rep3_B9_R2_trimmed.fastq.gz:md5,b0b5512e41d87ce28c8b58cdcf38a88f",
                "Mmus_BAL-TAL_RRTN_BSL_Rep4_B10_R1_trimmed.fastq.gz:md5,d31b2a4a3f9722430d01a9728ee2c857",
                "Mmus_BAL-TAL_RRTN_BSL_Rep4_B10_R2_trimmed.fastq.gz:md5,0c688523916b3dbe3bdf96bd32aa1968",
                "Mmus_BAL-TAL_RRTN_GC_Rep4_G10_R1_trimmed.fastq.gz:md5,d68d80608036afc6adcbfff4704eef89",
                "Mmus_BAL-TAL_RRTN_GC_Rep4_G10_R2_trimmed.fastq.gz:md5,880d7cf7a8a23c16e45370cf8c8f78a7"
            ],
            "STAR_NumNonZeroGenes.csv:md5,5efed758dc7c740905c48299f292fdbb",
            "STAR_Unnormalized_Counts.csv:md5,261f8731a7c0247217ea3ae2c73d5843",
            "Mmus_BAL-TAL_LRTN_BSL_Rep1_B7.genes.results:md5,ed4ca7a2d1352f82e0e97491212d40fa",
            "Mmus_BAL-TAL_LRTN_BSL_Rep1_B7.isoforms.results:md5,1a519ec4f3e8e65a6aabd05c298cd5d7",
            "RSEM_NumNonZeroGenes.csv:md5,866c1d9110c36a7ec9ee1eac93d3a1a3",
            "RSEM_Unnormalized_Counts.csv:md5,4aa50a767781e24ba1abed03adeda0e7",
            [
                "ERCC_Normalized_Counts.csv:md5,b1d9d5a546a23b6709a9f8c60548b6a7",
                "Normalized_Counts.csv:md5,b4ba348d5446f8ba546a46b966087c1b",
                "RSEM_Unnormalized_Counts.csv:md5,931c6070b5e19909929c5a217713500b"
            ],
            [
                [
                    "ERCCnorm_SampleTable.csv:md5,bfd18bbc7d34e41c23f0c9107f5d75c9",
                    "ERCCnorm_contrasts.csv:md5,66d74b686885ffd4eccdf55823c0e3ce",
                    "ERCCnorm_differential_expression.csv:md5,e713875ee762bccaacac2264f215d8b1",
                    "visualization_PCA_table_ERCCnorm.csv:md5,85edbf46c6af087cfe83e3a60f069a71",
                    "visualization_output_table_ERCCnorm.csv:md5,245d2ae218045f8c0a519c3c68119c73"
                ],
                "SampleTable.csv:md5,bfd18bbc7d34e41c23f0c9107f5d75c9",
                "contrasts.csv:md5,66d74b686885ffd4eccdf55823c0e3ce",
                "differential_expression.csv:md5,00cf45e546529c81c0a43ae1b8495a59",
                "visualization_PCA_table.csv:md5,5c461d35b12d5946c2105f705a03c6d3",
                "visualization_output_table.csv:md5,58578caedc33e6a0230ba80abe61f0d5"
            ],
            [
                "software_versions.md:md5,8946ba29cc457fe4d83687ee83253e14"
            ],
            "Mmus_BAL-TAL_LRTN_BSL_Rep1_B7.geneBodyCoverage.txt:md5,2bd21b4fa65d24275adab0e1273af337",
            "Mmus_BAL-TAL_LRTN_GC_Rep3_G9_infer_expt.out:md5,248ae9ba2b87a705ea046389342942a1",
            "Mmus_BAL-TAL_LRTN_FLT_Rep2_F7.inner_distance.txt:md5,f3984c71448fc339a5c1e8773dd67c34",
            "Mmus_BAL-TAL_RRTN_BSL_Rep4_B10_read_dist.out:md5,becb7190c85474e7f3ef07c6372491f6"
        ],
        "timestamp": "2023-01-25T23:04:11+0000"
    }
}